mono/mini/interp/interp.c

```diff
@@ -1596,7 +1573,6 @@ dump_stackval (GString *str, stackval *s, MonoType *type)
 	}
 }
 
-#if DEBUG_INTERP
```

Comment:
`dump_retval` is only used in code guarded by `DEBUG_INTERP`, so we should keep that to avoid unused warnings.

Reply:
it is already guarded

Reply:
oh! nice